### DISCUSSION TITLE
Why custom buttons no longer working after last update?

### DISCUSSION THREAD

**Discussion by ERYpTION** [osmandapp/OsmAnd#20585] [Q&A]
After the latest app update (OsmAnd+ 4.8.6) some of my custom buttons no longer work. I had set them up so that I can easily toggle the poi restroom on/off on the map. One button only has the restroom poi set while another (for another map profile) has both restroom and drinking water set (see screenshot below). This setup worked fine until the latest update. The button with both restroom and drinking water still show drinking water poi. Is this a bug or did something change with the restroom poi that affects the custom buttons?

![Screenshot_20240812-214827](https://github.com/user-attachments/assets/69dff739-e23c-4896-bb3b-583c763e0d7f)

**Comment by DmitryAlexei** [comment]
could you please provide you profile? go to menu > settings > your profile > export profile
To attach the profile to GitHub, please add .txt extension at the end of the file name.

**Reply by ERYpTION** [reply]
Like this?
[Vandring.osf.bin.txt](https://github.com/user-attachments/files/16600383/Vandring.osf.bin.txt)
[Vis kort.osf.bin.txt](https://github.com/user-attachments/files/16600384/Vis.kort.osf.bin.txt)

**Reply by ERYpTION** [reply]
Also (although probably not related) I have noticed that since the update the 'Trip recording' notification from my phones top pull-down bar (not sure what it's called) always show. Previously this would only show if I actually was or had used the recording feature.

**Reply by vshcherb** [reply]
Trip recording could be disabled in settings

**Reply by ERYpTION** [reply]
> in settings
 
In OsmAnd settings (I don't see any) or the device system settings? I don't want to disable it completely since the trip recording notification in the pull-down is useful when I actually record a track but now it shows all the time. I can swipe it away but it comes back as soon as I open the app again and stays. Also the is a persistent notification circle on the OsmAnd app-icon now.

I apologize for raising two problems in this thread. I thought perhaps they were connected since they both appeared after the latest app-update. The custom button issue is the one I don't understand at all and most want to solve.

**Reply by DmitryAlexei** [reply]
@ERYpTION Unfortunately your Quick action buttons were not included in export file.
Please resend export file with Quick actions and other related to profile settings (if they will be available) 

<table style="width:100%">
  <tr>
<th><img src="https://github.com/user-attachments/assets/dc00a0c1-e885-403e-b2b6-dd2d7cc38df8"  width="300" height="auto"/> </td>
  </tr>
</table>

**Reply by ERYpTION** [reply]
[Export_2024-08-14.osf.bin.txt](https://github.com/user-attachments/files/16611086/Export_2024-08-14.osf.bin.txt)
Better?

**Reply by ERYpTION** [reply]
By the way: I have tried using the POI Overlay to show the toilets but this also does not work.

**Reply by DmitryAlexei** [reply]
I've reproduced this bug and created a separate issue https://github.com/osmandapp/OsmAnd/issues/20617
- it seems, that issue only appears after update. So a complete reinstallation of the app may help. Also may help clearing app cash and data. Don't forget to create a backup before reinstall
- it is also recommended to change the rendering engine: menu > settings > OsmAnd settings > Map rendering engine > Version2

**Reply by ERYpTION** [reply]
Thank you. I will attempt a reinstall. Not so sure about using the Version2 rendering engine. I switched back to v1 to solve an issue with the app starting to repeatedly crash (also after an update).

**Reply by ERYpTION** [reply]
I tried reinstalling and the toilet poi issue went away. However, once I imported my backup with my profiles etc. it came back again after a short while. Not sure how to proceed.

**Comment by tommycrock** [comment]
FWIW I'm getting the trip recording notification is always on problem for the last week or two (i.e. even when I haven't started a recording and the app is in the background)

**Reply by DmitryAlexei** [reply]
You can disable notification in menu > plugins > trip recording > notification > disable
